tests/integrationv2/providers.py

```diff
@@ -416,6 +416,29 @@ def setup_server(self):
         return cmd_line
 
 
+class JavaSSL(Provider):
+    """
+    NOTE: Only a Java SSL client has been set up. The server has not been 
+    implemented yet.
+    """
+    def __init__(self, options: ProviderOptions):
+        self.ready_to_send_input_marker = None
+        Provider.__init__(self, options)
+    
+    def setup_server(self):
+        pytest.skip('JavaSSL does not support server mode at this time')
```

Comment:
Very cool. This is the setup I was hoping for. It will be neat to see which tests "just work" once support is added to the java client.